### PR TITLE
Added Inline CTA component

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Added CTA component
- Added CTA story
- Added test file
- Added new icon to /images folder

### CODE REVIEW

File: components/base/InlineCTA.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+`;
+const IconStyle = styled.div`
+  margin: 15px 33px 15px 15px;
+`;
+
+const TextStyle = styled.div`
+  height: 50px;
+  width: 506.84px;
+  color: ${PARAGRAPH_TEXT};
+  font-family: "Roboto", sans-serif;
```
Comment (sausman) [inline components/base/InlineCTA.tsx]: Let's use `${ROBOTO}` here

File: components/base/InlineCTA.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    width: auto;
+  }
+`;
+const IconStyle = styled.div`
+  margin: 15px 33px 15px 15px;
+`;
+
+const TextStyle = styled.div`
+  height: 50px;
+  width: 506.84px;
```
Comment (sausman) [inline components/base/InlineCTA.tsx]: Oddly specific @ShreyaThapa, lol

File: stories/InlineCTA.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import InlineCTA from "../components/base/InlineCTA";
+import guideIcon from "../public/images/guides-icon-large.svg";
+
+storiesOf("Inline CTA", module).add("Default", () => (
```
Comment (sausman) [inline stories/InlineCTA.stories.tsx]: Let's move this to `stories/base/` and prefix this with `base|` before merging.

File: components/base/InlineCTA.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "@emotion/styled";
+import arrowIcon from "../../public/images/arrow-right-orange.svg";
+import { GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
+import { BOX_SHADOW_5 } from "../shadowDepths";
+
+const BoxStyle = styled.div`
+  height: 78px;
```
Comment (sausman) [inline components/base/InlineCTA.tsx]: I think we could get away with removing `height`, `width` and the `@media` queries from a couple of these styled components and just adding a `max-width` here:

```diff
diff --git a/components/base/InlineCTA.tsx b/components/base/InlineCTA.tsx
index a982ec3..b314af4 100644
--- a/components/base/InlineCTA.tsx
+++ b/components/base/InlineCTA.tsx
@@ -5,8 +5,7 @@ import { GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
 import { BOX_SHADOW_5 } from "../shadowDepths";
 
 const BoxStyle = styled.div`
-  height: 78px;
-  width: 656px;
+  max-width: 656px;
   box-sizing: border-box;
   border: 1px solid ${GREY_2};
   border-radius: 5px;
@@ -17,28 +16,18 @@ const BoxStyle = styled.div`
   :hover {
     box-shadow: ${BOX_SHADOW_5};
   }
-  @media only screen and (max-width: 656px) {
-    height: auto;
-    width: auto;
-  }
 `;
 const IconStyle = styled.div`
   margin: 15px 33px 15px 15px;
 `;
 
 const TextStyle = styled.div`
-  height: 50px;
-  width: 506.84px;
   color: ${PARAGRAPH_TEXT};
   font-family: "Roboto", sans-serif;
   font-size: 15px;
   font-weight: 500;
   line-height: 25px;
   margin: 14px auto;
-  @media only screen and (max-width: 656px) {
-    height: auto;
-    width: auto;
-  }
 `;
 
 const ArrowStyle = styled.div`
```

File: components/base/InlineCTA.tsx (hunk 2 above)
Comment (ShreyaThapa) [reply components/base/InlineCTA.tsx]: lol yeah, I was going off of the design mockups pixel to pixel, which I now understand does not have to be the case. haha

**Review by sausman — APPROVED** [review]

**Pull request merged**